packages/test-helpers/src/asserts/assertBn.js

```diff
@@ -1,9 +1,10 @@
 const { assert } = require('chai')
+const { bn } = require('../')
 
 const assertBn = (actual, expected, errorMsg) => {
   assert.equal(
-    actual.toString(),
-    expected.toString(),
+    bn(actual).toString(),
+    bn(expected).toString(),
```

Comment:
Bear in mind this may not work for different versions of BN, I prefer being flexible here. Calling `.toString()` to any type of BN seems safer to me.

Reply:
Done in 9bebc4d.